README.md

```diff
@@ -43,4 +43,4 @@ To start in development mode, run
 yarn start
 ```
 
-To contribute to the source, please read [CONTRIBUTING.md](https://github.com/PastelNetwork/Pastel_Electron_Wallet/blob/master/docs/CONTRIBUTING.MD).
+To contribute to the source, please read [CONTRIBUTING.md](https://github.com/PastelNetwork/Pastel_Electron_Wallet/blob/master/docs/CONTRIBUTING.md).
```

Comment:
If you try to navigate to https://github.com/PastelNetwork/Pastel_Electron_Wallet/blob/master/docs/CONTRIBUTING.md in the browser, you will see that this link is broken. The previous one was correct (with an uppercase extension).

Reply:
This PR also changes the filename from uppercase to lowercase. The link should work after merged.

Reply:
@namnm Sorry it took long, I don't think the above is true. If you look at the file structure of this branch (https://github.com/pastelnetwork/pastel-electron-wallet/tree/hotfix/mistyped-extension-contributing/docs), you will see that `CONTRIBUTING.MD` still has uppercase extension

Reply:
@br4in3x Thanks, I have just used `git mv` to perform the update